internal/experiment/generation/stormforger.go

```diff
@@ -235,6 +235,10 @@ func (s *StormForgerSource) stormForgerAccessToken(org string) *optimizeappsv1al
 			accessToken.SecretKeyRef.Key = org
 		}
 
+		if accessToken.Literal != "" {
+			accessToken.Literal = strings.TrimSuffix(accessToken.Literal, "\n")
```

Comment:
I'd go with:
```
accessToken.File = strings.TrimSpace(accessToken.File)
accessToken.Literal = strings.TrimSpace(accessToken.Literal)
```
(the conditional is over optimized, trim all leading/trailing whitespace, get both the literal and file name just to be safe).